docs/docs/notification-center/react-components.md

```diff
@@ -173,3 +173,147 @@ Then pass the created HMAC to your client side application forward it to the com
 <NovuProvider subscriberId={'PLAIN_TEXT_ID'} subscriberHash={'HASHED_SUBSCRIBER_ID'} applicationIdentifier={'APP_ID'}>
 </NovuProvider>
 ```
+
+## Customizing the notification center theme
+
+The notification center component can be customized by passing a `theme` prop to the `PopoverNotificationCenter` component.
+
+```tsx
+const theme: INovuTheme = {
+   dark: {
+      // Dark Theme Props
+   },
+   light: {
+       // Light Theme Props
+   },
+   common: {
+      // Common
+   }
+};
+
+<PopoverNotificationCenter theme={theme}>
+  {({ unseenCount }) => <NotificationBell unseenCount={unseenCount} />}
+</PopoverNotificationCenter>
+
+export interface INovuTheme {
+  layout?: IThemeLayout;
+  header?: IThemeHeader;
+  popover?: IThemePopover;
+  notificationItem?: IThemeNotificationListItem;
+  footer?: IThemeFooter;
+  loaderColor?: string;
+  unseenBadge?: IThemeUnseenBadge;
+}
+
+```
+
+A theme object can be used to customize the notification center's layout, header, popover, notification list item, footer, and unseen badge. 
+The object can be modified partially or completely, depending on the level of customization you want to achieve.
+
+Here are the optional fields that can be used to customize the notification center:
+
+A table of IThemeLayout properties:
+
+## Default styles for `IThemeLayout`
+
+| Property | Default Value - Light Theme              | Default Value - Dark Theme |
+| -------- |------------------------------------------|----------------------------|
+| `background` | `#FFFFFF`                                | `#1E1E26`                                |  
+| `boxShadow` | `0px 5px 15px rgba(122, 133, 153, 0.25)` | `0px 5px 20px rgba(0, 0, 0, 0.2)` |
+| `borderRadius` | `7px`                                     |`7px`                                         |
+| `wrapper.secondaryFontColor` | `#BEBECC`                                | `#525266`                               |
+
+```tsx
+export interface IThemeLayout {
+  background?: string;
+  boxShadow?: string;
+  borderRadius?: string;
+  wrapper?: {
+    secondaryFontColor?: string;
+  };
+}
+```
```

Comment:
Those ones can be removed since we have them in the table :)

Reply:
You mean remove the all the tsx code? Because then we should also put type in the table

Reply:
It might make the table look awkward because boxShadow and the linear values are quite long

Reply:
I think we can remove all the TSX codes, I think from the value it's quite obvious that it's a string. But we can also add another column if you think it will be useful.